controlplane/kubeadm/internal/workload_cluster_etcd.go

```diff
@@ -116,9 +129,10 @@ func (w *Workload) EtcdIsHealthy(ctx context.Context) (HealthCheckResult, error)
 		} else {
 			unknownMembers := memberIDSet.Difference(knownMemberIDSet)
 			if unknownMembers.Len() > 0 {
+				inconsistentMemberList = true
```

Comment:
if we are changing the variable name, this should be
```suggestion
				nodesResportSameMemberList = false
```